pages/what-are-sprints.md

```diff
@@ -0,0 +1,32 @@
+title: What on earth is a code sprint?
+---
+
+Most of us who work on open-source projects collaborate at a distance. A code sprint at a conference is a chance to get
+together and talk and work in person.
+
+## What happens at a sprint?
+
+It’s a number of developers - programmers, designers, documentation writers - together in a room, talking to each other, or sitting at their computers, working on their own, or in pairs, or discussing problems in groups. Typically, you’ll select a ticket (representing a bug, or an improvement that needs to be made) from the project's issue tracker, and get to work on that. As you work, you might want to check with a more experienced coder whether you’re going about it the right way and get feedback on what you’re doing.
+
+## So can I join in?
+Absolutely. Everyone’s welcome. It’s not an elite club, or for experts only. All of the projects that will be running sprints welcome anyone who wants to help.
+
+## I’ve never taken part in a sprint before though.
+That’s no problem. There’ll be an introduction to explain how it works, and other people will be only too pleased to help you get started or to work with you.
+
+## What do I need?
+You'll need to know the basics of using the appropriate version control system (it'll probably be Git/GitHub) to check out code and make pull requests, to know how to use the issue tracker, and so on. You’ll also need your own computer with a suitable Python environment set up on it - but there'll be people to help with that too.
+
+## But I’m not actually a very good Python programmer.
+That’s OK too, neither am I and neither is everyone else. You truly don’t need to be very good Python programmer. In fact, you barely need to be a programmer at all, because even the most novice sprinter can make a very valuable contribution by helping improve documentation, for example.
+
+## I’d be worried about holding up everyone else though.
+It simply doesn’t work like that. For one thing, one of the main points of a sprint is to encourage and help new people. You’ll be able to work at your own pace, learning the things you need to learn as you go along, and you’ll be supported by other people.
+
+## Do I have to stay for a whole sprint?
+Not at all. Drop in just for a bit, if you just want to see what it’s like. There'll be plenty of other things on the code day too, such as [clinics and workshops](/code-day/), so don’t miss them. Other people will be dropping in and out of sprints as well.
```

Comment:
I expect we will, but we don't have anything timetabled yet.

I think we can simply drop that second-last sentence for now @alexwlchan .